enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -20,6 +20,34 @@ or using yarn:
 yarn add @metabase/embedding-sdk-react
 ```
 
+## Which SDK version to use
+
+The SDK version tracks with the Metabase version.
+
+So, for example, if you're on Metabase version `1.51.1`, you should use the SDK `1.51`.
```

Comment:
Is this the correct advice?

Reply:
I'm not sure if the point version would make it too confused when there are newer version or not. Do you think phrasing it like this would be better?

```suggestion
So, for example, if you're on Metabase version `1.51.x`, you should use the SDK with the tag `51-stable`.
```